doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+---------------------------------
+
+`Plutus ledger constraints <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger-constraints>`_ 
+contains a constraints-based API that can be used to generate on-chain validation 
+functions and to build transactions by providing a list of constraints. The main 
+design is to be able to use the same constraints on-chain and off-chain in a plutus 
+application. The off-chain part generates transactions based on types in plutus-ledger. 
+
+For example:
+
+   * ``checkScriptContext (MustSpendAtLeast 10Ada, MustProduceOutput myOutput, …)``
+   * ``mkTx (MustSpendAtLeast 10Ada, MustProduceOutput myOutput, …)``
+
++--------------+---------------------------------------------------------------+
+| Plutus ledger constraints                                                    |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-ledger-constraints``                     |
++--------------+---------------------------------------------------------------+
+| Dependencies | plutus-ledger                                                 |
++--------------+---------------------------------------------------------------+
+| State        | In the process of being deprecated in favor of                |
+|              | plutus-tx-constraints in order to build transactions using    |
+|              | cardano-api instead of plutus-ledger.                         |
++--------------+---------------------------------------------------------------+
+
+5. Plutus Tx constraints
+---------------------------------
+
+`Plutus-tx-constraints <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-tx-constraints>`_ 
+contains a constraints-based API that can be used to generate on-chain validation 
+functions and to build transactions by providing a list of constraints. The main 
+design is to be able to use the same constraints on-chain and off-chain in a Plutus 
+application. The off-chain part generates transactions based on types in cardano-api. 
+
++--------------+---------------------------------------------------------------+
+| Plutus Tx constraints                                                        |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-tx-constraints``                         |
++--------------+---------------------------------------------------------------+
+| Dependencies | plutus-ledger-api, plutus-script-utils, cardano-api           |
++--------------+---------------------------------------------------------------+
+| State        | Work in progress to support all the functionality from        | 
+|              | plutus-ledger-constraints.                                    |
++--------------+---------------------------------------------------------------+
+
+6. Plutus chain index
+---------------------------------
+
```

Comment:
Soon to be deprecated in favor of Marconi

Reply:
I wouldn't even include it.